src/api/services/usersService.ts

```diff
@@ -4,9 +4,17 @@ import { z } from 'zod';
 import { Participant } from '../entities/Participant';
 import { User, UserDTO } from '../entities/User';
 import { getLoggers } from '../helpers/loggingHelpers';
+import { isUserAnApprover } from './approversService';
+
+export type UserWithIsApprover = User & { isApprover: false };
 
 export const findUserByEmail = async (email: string) => {
-  return User.query().findOne('email', email);
+  const user = await User.query().findOne('email', email);
```

Comment:
This is causing a bunch of typescript errors for me - before, the return type was `User | undefined`, but because this is using the spread operator (below) to combine user with isApprover, it changes it to always returning something, but all of the properties are now `string | undefined`, 'number | undefined`, etc.

Reply:
have tried to address - let me know if that works